src/vnodeTransformers/stubComponentsTransformer.ts

```diff
@@ -166,11 +166,13 @@ export function createStubComponentsTransformer({
       if ('transition' in stubs && stubs['transition'] === false) return type
       if ('Transition' in stubs && stubs['Transition'] === false) return type
 
-      return createStub({
-        name: 'transition',
-        type,
-        renderStubDefaultSlot: true
-      })
+      if (stubs['transition'] === true || stubs['Transition'] === true) {
```

Comment:
If this is the proper way to fix this (I don't remember how it works exactly), then the same fix must be applied to transition-group, teleport and the other cases in the function no?

Reply:
Oh yeah was just being lazy because this is the thing that affects me 😅 Will change those as well (it's a shame there's not an easy way to target both PascalCased and kebabe-cased, but that's beyond the scope of this change...)